config.cc

```diff
@@ -32,6 +32,18 @@ namespace Config {
 
 namespace
 {
+#ifdef XDG_BASE_DIRECTORY_COMPLIANCE
+  const char xdgSubdirName[] = "goldendict";
+  QDir getDataDir()
+  {
+    QDir dir = QStandardPaths::writableLocation( QStandardPaths::GenericDataLocation );
+    dir.mkpath( xdgSubdirName );
+    if ( !dir.cd( xdgSubdirName ) )
+      throw exCantUseDataDir();
+
+    return dir;
+  }
+#endif
```

Comment:
Please separate the `#endif` and the next function declaration with a blank line.